lib/open_food_network/payments_report.rb

```diff
@@ -124,7 +124,7 @@ def columns
              }.sum(&:amount)
            }
          },
-         proc { |orders| orders.sum(&:outstanding_balance) }]
+         proc { |orders| orders.sum { |order| order.outstanding_balance.to_f } }]
```

Comment:
Can't this be simplified again?

Reply:
Yes, I missed it. I amended the last commit to include this as well.